components/SocialButton/SocialButton.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Button from '../Button';
+import FacebookIcon from './sub-components/FacebookIcon';
+import GoogleIcon from './sub-components/GoogleIcon';
+import { theme as defaultTheme } from '../shared';
+
+const BUTTON_SIZE = '40px';
+const BUTTON_PADDING = defaultTheme.spacing.xsmall;
+
+const BaseButton = styled(Button)`
+  width: ${BUTTON_SIZE};
+  height: ${BUTTON_SIZE};
+  border-radius: 9px;
+  cursor: pointer;
+  overflow: hidden;
+  outline: none;
+`;
+
+const GoogleButton = styled(BaseButton)`
+  padding: ${BUTTON_PADDING - 1}px;
+`;
+
+const FacebookButton = styled(BaseButton)`
+  border: none;
+  background-color: #3c5193;
+  padding: ${BUTTON_PADDING}px;
+`;
+
+const SocialButton = ({ provider, ...props }) => {
+  switch (provider) {
+    case 'facebook':
+      return (
+        <FacebookButton {...props}>
+          <FacebookIcon title="facebook-button" size="24" />
+        </FacebookButton>
+      );
+    case 'google':
+      return (
+        <GoogleButton {...props} stroked skin="neutral">
+          <GoogleIcon title="google-button" size="24" />
+        </GoogleButton>
+      );
+    default:
+      return <Button />;
+  }
+};
+
+SocialButton.propTypes = {
```

Comment:
Thinking a little more, how about we put a size prop (not required) here too?
and that `size`, via this prop, passes to `GoogleIcon` and `FacebookIcon`?
That way, the size does not be in static way and who implements has this control.

Ps: we could keep the 24px size by default in Proptypes.

What do you think?

Reply:
I agree with you, that's the ideal. But I thought that it should come in other versions of the component since changing the icon size causes breaks in the button layout, so I think is better provide pre-defined sizes, like "small", "large", etc